src/Components/DateInputForm/DateInputForm.tsx

```diff
@@ -0,0 +1,18 @@
+﻿import React from "react";
+import "./DateInputForm.scss"
+
+
+export function DateInputForm() {
+    return (
+        <div >
+            <form className="form">
+                <label>DATE: 
+                    <br/>
+                    <input type="date" id="date" name="date"
+                           min="12-08-05" max="2020-03-03" />
```

Comment:
Looks like this was true yesterday, but is a little less valid today! 

If we want a `max` field, then I think we are going to need to set it dynamically instead of hard coding it.
Another other option would be to validate the form in JS when it gets submitted.

The final option is to not bother validating it and see what happens! :) (would need good error handling when returning images)